docs/reference/postgresql12_migration.md

```diff
@@ -0,0 +1,133 @@
+| **Downtime Impact**      | Requires application downtime during migration | Minimal downtime |
+| **Primary Keys requirement** | No primary keys required; exports full table data regardless of constraints | Requires primary keys (or unique indexes) on tables|
+| **Schema compatibility**  | Allows schema modifications before restoration; supports cross-version migration | Requires schema compatibility between source and target databases |
+| **Performance**          | Can be slow for large datasets due to full export/import | More efficient for continuous updates, but may add replication overhead |
+
+### Logical Dump and Restore
+
+This option is recommended for **smaller databases** and **one-time migrations**.
+{.power-number}
+
+1. Provision a new PostgreSQl database in Percona Everest.
+
+
+
+2. Perform a logical dump of the old database:
+
+    - Use [pg_dump or pg_dumpall](https://www.postgresql.org/docs/current/app-pgdump.html) to **export** your data from PostgreSQL 12.
+
+        ??? example "Example: Source database (PostgreSQL 12)"
+            Run the following command on the source database (PostgreSQL 12):
+
+            ```sh
+            pg_dump -Fc -h <old-db-host> -U <user> <db_name> -f dump_file.dump
+            ```
+
+3. Restore data to the new Percona Everest database:
+
+    - Use [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) or [psql](https://www.postgresql.org/docs/current/app-psql.html) to import data into the newly created Percona Everest cluster.
+
+        ??? example "Example: Target database"
+            Run the following command on the target PostgreSQL database:
+        
+            ```sh
+            pg_restore -U your_user -h new_db_host -d new_db_name -F c your_db_backup.dump
+            ```
+
+4. Verify post-migration performance:
+
+    - Run queries to confirm [data integrity](https://www.postgresql.org/docs/current/applevel-consistency.html) and ensure compatibility.
+
+For comprehensive information, dive deep into the [PostgreSQL documentation on pg_dump](https://www.postgresql.org/docs/current/backup-dump.html).
+
+### Logical Replication
+
+This option is recommended for **minimal downtime**.
+{.power-number}
+
+1. Use [logical replication](https://www.postgresql.org/docs/current/logical-replication.html) to continuously replicate data from your PostgreSQL 12 cluster into a PostgresQLG 13+ cluster with little to no downtime.
+
+    - Enable logical replication on the source PostgreSQL 12 DB (ensure `wal_level = logical`).
+
+    ```sh
+    wal_level = logical
+    max_replication_slots = 10
+    max_wal_senders = 10
+    ```
+
+2. Set up replication slots and publications on the source:
+
+    - Connect to PG 12 and create a slot.
+
+        ```sql
+        SELECT pg_create_logical_replication_slot('my_slot', 'pgoutput');
+        ```
+
+3. Set up the subscription on the target database:
+
+- Connect to the target database and create a subscription.
```

Comment:
Here's the issue. Move this line to be a nested item under 3.

Reply:
Ok